PR for using ngx_http_upstream_ketama_chash as dynamic module

heyadityak opened the pull request:
This PR is created with referece to #1 . Please review & merge the same. If there are any questions, please let me know.

heyadityak committed:
Updates documentation and config to allow dynamic module use

flygoast reviewed (changes requested):
Thanks for promoting.

Maybe need add `ngx_module_libs="-lm"` to file `config`.

And at README.md, you can display static and dynamic installation methods. Like this: 
https://github.com/nginx-modules/ngx_http_hmac_secure_link_module

Thanks.

heyadityak committed:
Adds the requested changes; updates docs

This change will add the library modules option in the config as well as add documentation steps for using both as a static & dynamic module

heyadityak commented:
Hi @flygoast I have added my changes. Please review the same.
